### DISCUSSION TITLE
More Data/Columns in Voucher Overview

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2592] [Ideas] [answered]
Hey everyone,

since we're using vouchers as a way to reserve spots for "VIP"s it would be nice to add a little more info to the voucher overview.
I understand this is a tricky task, since when generated in bulk and set out via email like in our case, the only field holding additional information is the comment field, which may not really be suitable to be displayed in a column. Displaying the associated email in a column sounds fair to me, but it's not really elegant.
Looking at the voucher overview without an idea who redeemed his voucher and who hasn't (to manually trigger people to do it) is a bit lackluster.

Any ideas welcome, since - besides displaying emails which may be "superbunny420@yahoo.com" - there is no good solution yet. Creating a Tag, for every associated name isn't very elegant either.

Idea: Add another row to the CSV import for vouchers for "Display Name" or something.

Thanks,
Alex

**Comment by raphaelm** [comment, marked as answer]
Hm, I think using tags and comments would be the way to go currently.

**Reply by choseh** [reply]
mh, I really consider tags as an instrument for grouping vouchers, if you think grouping in groups of 1 is okay, then so be it ;)
